sharepoint/sharepoint-server-ps/sharepoint-server/New-SPWebApplication.md

```diff
@@ -41,7 +43,7 @@ For permissions and the most current information about Windows PowerShell for Sh
 New-SPWebApplication -Name "Contoso Internet Site" -Port 80 -HostHeader sharepoint.contoso.com -URL "http://www.contoso.com" -ApplicationPool "ContosoAppPool" -ApplicationPoolAccount (Get-SPManagedAccount "DOMAIN\wa")
 ```
 
-This example creates a new Web application by using an internal host header of sharepoint.contoso.com and a public URL of http://www.contoso.com.
+This example creates a new Web application by using an internal host header of sharepoint.contoso.com and a public URL of https://www.contoso.com.
```

Comment:
```suggestion
This example creates a new web application by using an internal host header of sharepoint.contoso.com and a public URL of https://www.contoso.com.
```

Reply:
The example and its description need to match, so if we're updating the description to use an SSL URL, then the example needs to be updated too.  But the example is also using port 80, which is an HTTP port rather than an HTTPS port.